src/test/java/catalinaTema5/Heroku/WebPageExample.java

```diff
@@ -0,0 +1,29 @@
+package catalinaTema5.Heroku;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebPageExample {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/basic-web-page-test.html");
+
+
+        WebElement firstLink = driver.findElement(By.id("basicpagetest"));
```

Comment:
This method throws an exception. Please double check how you identify the elements and make it pass. Thanks!